.github/workflows/generate-queries.yml

```diff
@@ -0,0 +1,30 @@
+name: Generate queries
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  master:
+    name: Generate Queries
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node.js 16
+        id: Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.14.0
```

Comment:
Question (non-blocking): Is it possible to grab the node version from the package.json rather than hard code it?

Reply:
Package specifies a range not a specific version so not really. We should move this to matrix at some point but for now let's see if this provides any value before we do that :+1: